README.md

```diff
@@ -13,7 +13,7 @@ how to build your own Terraform from scratch.
 
 ```terraform
 module "jvs" {
-  source     = "git::https://github.com/abcxyz/jvs.git//terraform/e2e?ref=main" # this should be pinned to the SHA desired
+  source     = "git::https://github.com/abcxyz/jvs-plugin-jira.git//terraform/example?ref=main" # this should be pinned to the SHA desired
```

Comment:
It is better for users to copy and paste the example module instead of use the example module as a source module, nested modules (SHA needs to pinned multiple places) are less preferred.

Reply:
+1. We should not encourage people to reuse this module. We can simply say go look at the terraform code in this folder as an example (of how to hook up JVS with JIRA plugin)

Reply:
SG, hardcode all the values in the example module instead